pwa/src/templates/documentation/installationDocumentation/InstallationDocumentationTemplate.tsx

```diff
@@ -0,0 +1,184 @@
+import * as React from "react";
```

Comment:
Feedback regarding the entire text:

1. We're using "u" and "je" inconsistent; please check with @rubenvdlinde which is preferred and
2. This is a lot of documentation about the Skeleton Application, is this even relevant? Open catalogi is an implementation of the Skeleton Application, sure. But is this much in-depth documentation relevant? I think not, please also check with @rubenvdlinde .